spec/requests/rack_attack_spec.rb

```diff
@@ -34,6 +58,14 @@ def unique_user_params
       end
     end
 
+    it "response to the next attempt from the same IP includes retry-after header" do
+      post user_session_path, params: unique_user_params.to_query, env: { "REMOTE_ADDR" => ip }
+      expect(response).to have_http_status(:too_many_requests)
+      expect(response.headers["Retry-After"]).not_to be_nil
+      expect(response.headers["Retry-After"].to_i).to be > 0
+      expect(response.headers["Retry-After"].to_i).to be <= ArchiveConfig.RATE_LIMIT_PERIOD
```

Comment:
I think this and line 100 both need to use `RATE_LIMIT_LOGIN_PERIOD`.

Reply:
I think you are correct! Done.